sdk/keyvault/azure-keyvault-certificates/migration_guide.md

```diff
@@ -27,7 +27,7 @@ To try and improve the development experience across Azure services, a set of un
 
 ### Cross Service SDK improvements
 
-The modern Key Vault client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as 
+The modern Key Vault client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
```

Comment:
I updated these links yesterday but didn't realize that everyone had the same issue -- on line 26 I used https://azure.github.io/azure-sdk/general_introduction.html, but that can be replaced with https://azure.github.io/azure-sdk/python/guidelines/index.html. (This goes for keys and secrets as well)

Reply:
Adding this change